components/brave_rewards/resources/tip/lib/interfaces.ts

```diff
@@ -2,36 +2,15 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-type EmptyMediaData = {
-  mediaType: 'none'
-}
-
-export type TwitterMediaData = {
-  mediaType: 'twitter'
+export type MediaMetaData = {
```

Comment:
I was thinking about this before I merged and I would prefer to keep separate interface types for the different media types, even if it results in some duplication. A good example of how this can help is GH: since there is no "post" info delivered into the tip dialog, we want TypeScript to check that we don't accidentally rely on "postText" if we are dealing with a "github" media type.

Also, I didn't notice before but I think we can remove the "export" from `TwitterMediaData`.

Reply:
OK gotcha, hadn't realized you were going in that direction. Makes total sense, will adjust it.